src/engraving/libmscore/measure.cpp

```diff
@@ -4112,12 +4008,67 @@ static bool hasAccidental(Segment* s)
 }
 
 //---------------------------------------------------------
-//   computeMinWidth
-//    sets the minimum stretched width of segment list s
-//    set the width and x position for all segments
+//      Stretch formula
+//      Computes the stretch for a given segment depending
+//      on its duration with respect to the shortest one.
+//      Three different options proposed (see documentation).
 //---------------------------------------------------------
+float Measure::durationStretch(Fraction curTicks, const Fraction minTicks) const
+{
+    qreal userSlope = userStretch() * score()->styleD(Sid::measureSpacing);
+    static constexpr qreal baseSlope = 0.647;
```

Comment:
explanation comment needed

stick to this rule every time you want to write hardcoded numbers

Reply:
This one is explained in the longer comment that follows